builder/src/build.ts

```diff
@@ -199,7 +199,7 @@ export namespace Build {
       themeConfig.push({
         mode: 'production',
         entry: {
-          index: path.join(name, themePath)
+          index: path.join(packagePath, themePath)
```

Comment:
For consistency, can we use `packageDir`, which is used above? I know they should be the same, but still, for consistency with usage above.

Reply:
Should we remove one of the variables to reduce confusion? If they are always the same.